### ISSUE TITLE
Sequential simulations error on second step

### ISSUE THREAD

**Issue by TragerJoswig-Jones** [NREL-SIIP/PowerSimulations.jl#499]
Modifying the SIIPExample '2_sequential_simulations.jl' script to have more than one step in the simulation results in an error on the second step.

```
sim = Simulation(
    name = "rts-test",
    steps = 2, # Changed from 1 to 2
    stages = stages_definition,
    stages_sequence = DA_RT_sequence,
    simulation_folder = rts_dir,
)
```

---

```
julia> include("2_sequential_simulations.jl")
[ Info: Precompiling PowerSimulations [e690365d-45e2-57bb-ac84-44ba829e73c4]
[ Info: Precompiling Cbc [9961bab8-2fa3-5c5a-9d89-47fab24efd76]
[ Info: Parsing csv data in branch.csv ...
```
...
```
Executing Step 2
ERROR: LoadError: UndefVarError: status_remains_off not defined
Stacktrace:
 [1] calculate_ic_quantity(::PowerSimulations.ICKey{DevicePower,ThermalStandard}, ::InitialCondition{ParameterJuMP.ParameterRef}, ::Float64, ::Dict{PowerSimulations.CacheKey,PowerSimulations.AbstractCache}) at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\update_initial_conditions.jl:91
 [2] initial_condition_update!(::Stage{GenericOpProblem}, ::PowerSimulations.ICKey{DevicePower,ThermalStandard}, ::Array{InitialCondition,1}, ::InterStageChronology, ::Simulation) at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:663
 [3] _update_initial_conditions!(::Stage{GenericOpProblem}, ::Simulation) at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:807
 [4] _update_stage! at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:832 [inlined]
 [5] update_stage!(::Stage{GenericOpProblem}, ::Simulation) at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:843
 [6] macro expansion at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:925 [inlined]
 [7] macro expansion at C:\Users\Trager\.julia\packages\TimerOutputs\dVnaw\src\TimerOutput.jl:190 [inlined]
 [8] macro expansion at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:924 [inlined]
 [9] macro expansion at C:\Users\Trager\.julia\packages\TimerOutputs\dVnaw\src\TimerOutput.jl:190 [inlined]
 [10] macro expansion at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:906 [inlined]
 [11] macro expansion at C:\Users\Trager\.julia\packages\TimerOutputs\dVnaw\src\TimerOutput.jl:190 [inlined]
 [12] macro expansion at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:892 [inlined]
 [13] macro expansion at C:\Users\Trager\.julia\packages\TimerOutputs\dVnaw\src\TimerOutput.jl:190 [inlined]
 [14] _execute!(::Simulation; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:889
 [15] _execute! at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:880 [inlined]
 [16] #80 at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:871 [inlined]
 [17] with_logstate(::PowerSimulations.var"#80#81"{Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}},Simulation}, ::Base.CoreLogging.LogState) at .\logging.jl:398
 [18] with_logger at .\logging.jl:505 [inlined]
 [19] execute!(::Simulation; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:870
 [20] execute!(::Simulation) at C:\Users\Trager\.julia\packages\PowerSimulations\7yuhI\src\core\simulation.jl:866
 [21] top-level scope at C:\RealLab\PRAS_Storage\SIIP\SIIPExamples.jl-master_8.3\test\PowerSimulations_examples\2_sequential_simulations.jl:71
 [22] include(::String) at .\client.jl:439
 [23] top-level scope at REPL[2]:1
in expression starting at C:\RealLab\PRAS_Storage\SIIP\SIIPExamples.jl-master_8.3\test\PowerSimulations_examples\2_sequential_simulations.jl:71
```

**Comment by jd-lara** [comment]
This should be fixed in v0.6.1 can you confirm which version are you using?

**Comment by TragerJoswig-Jones** [comment]
> This should be fixed in v0.6.1 can you confirm which version are you using?

Ah the environment I downloaded had v0.6.0. After updating to v0.6.1 the error no longer occurs. Thank you!
